site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -188,7 +204,7 @@ include::_tenant-generator-type-config-rules.adoc[]
 [field]#tenant.externalIdentifierConfiguration.passwordlessLoginTimeToLiveInSeconds# [type]#[Integer]# [required]#Required# [since]#Available since 1.8.0#::
 The time in seconds until a passwordless code is no longer valid and cannot be used by the Passwordless API. Value must be greater than 0.
 
-[field]#tenant.externalIdentifierConfiguration.registrationVerificationIdGenerator.length# [type]#[Integer]# [required]#Required# [since]#Available since 1.8.0#::
+[field]#tenant.externalIdentifierConfiguration.registrationVerificationIdGenerator.length# [type]#[Integer]# [optional]#Optional# [default]#6# [since]#Available since 1.8.0#::
```

Comment:
Was this always optional, or did it become optional in a recent release? If the latter, we should note.

Reply:
I think this was copy pasta 🙂